docs/index.rst

```diff
@@ -82,7 +82,7 @@ Packages
     :maxdepth: 1
     :caption: Examples
 
-    Getting Started <tutorials/notebooks/point_clouds>
+    Getting Started <tutorials/notebooks/basic_ot_between_datasets>
```

Comment:
this means the `point_cloud` note book is removed completely, but I think it had its own use to highlight other functionalities (such as defining alternative cost functions)